app/components/templates/templates.html

```diff
@@ -21,13 +21,13 @@
 
         <form class="form-horizontal">
           <!-- description -->
-          <div ng-if="state.selectedTemplate.Note">
+          <div ng-if="state.selectedTemplate.Note || state.selectedTemplate.DeprecatedFormat">
```

Comment:
Can you remove the `|| state.selectedTemplate.DeprecatedFormat` in the comparison ?